public/js/simpleStrings-0.0.1.js

```diff
@@ -34,10 +34,13 @@
         var text = item.clone().children().remove().end().text();
 
         if(matches = text.match(/{{([^}]*)}}/) ) {
+          matches.shift();
           $.each(matches, function() {
               var keyword = this;
-              if(keyword in settings.strings) {
-                  text = text.replace('{{'+keyword+'}}', settings.strings[keyword]);
+              var newText = keyword.split('.')
+                    .reduce((o, i) => o[i], settings.strings);
```

Comment:
I'm going to guess that this eliminates IE support. Can we use the `.reduce(index, obj)` form instead?

Reply:
There, I fixed it.